### PR TITLE
Feature/remove submodule

### PR DESCRIPTION
**Pull request by lats1**

### CODE REVIEW

File: README-develop.md [hunk 1]
```diff
@@ -20,6 +15,28 @@ open http://$(docker-compose port nginx 80)
...
 ## Setup for development
 
+Add core-bundle to the project:
+```sh
+git clone git@github.com:aakb/kontrolgruppen-core-bundle.git bundles/core-bundle
+```
+
+Change configuration in composer.json to use the local copy of the core-bundle: (make sure that the version constraint of the package is set to the current branch your local clone of the core bundle)
+Just make sure you don't commit theese changes, as they only are for the local setup.
+```json
```
Comment (rimi-itk) [inline README-develop.md]: I think we should show console commands for making these changes:

```sh
composer config repositories.kontrolgruppen/core-bundle path bundles/core-bundle
composer require 'kontrolgruppen/core-bundle:dev-develop'
```
(symlinking is the default if possible, cf. https://getcomposer.org/doc/05-repositories.md#path)

Maybe we should even remove the `json` snippet …

File: README.md [hunk 2]
```diff
@@ -28,19 +28,10 @@ See `https://ckeditor.com/latest/samples/toolbarconfigurator/index.html#advanced
...
-The composer merge plugin should be added under the require-dev section in the main composer file, so it will only
-run when --no-dev is not present when running composer install.
-
-This will replace the initializing and updating of submodules when deploying to production, because the required kontrolgruppen
-package will be fetched from vcs instead.
-
 ## Release procedure.
 
+Make sure that the composer.json is using the correct version constraint for the core-bundle, and that the repository for the core-bundle is set to the vcs-type and the url is the git repository.
+
```
Comment (rimi-itk) [inline README.md:38]: Maybe add something like

```sh
composer config repositories.kontrolgruppen/core-bundle vcs git@github.com:aakb/kontrolgruppen-core-bundle.git
```
(do we also need `composer require 'kontrolgruppen/core-bundle:«some branch name»'` here?)

and/or something like

> Run `git checkout --patch composer.json` to revert your develop changes to `composer.json`.

File: package.json [hunk 3]
```diff
@@ -53,14 +53,14 @@
         "dev": "encore dev",
         "watch": "encore dev --watch",
         "build": "encore production --progress",
-        "check-coding-standards-stylelint": "stylelint --config=.stylelintrc.js 'bundles/kontrolgruppen-core-bundle/Resources/assets/**/*.scss'",
+        "check-coding-standards-stylelint": "stylelint --config=.stylelintrc.js 'vendor/kontrolgruppen/core-bundle/Resources/assets/**/*.scss'",
```
Comment (rimi-itk) [inline package.json:56]: I think, personally, that this should be moved to the `core-bundle`'s own `package.json`, but we'll leave it like this for now.

**Review by rimi-itk — CHANGES_REQUESTED** [review]
Looks good. A few comments have been added.

File: README-develop.md [hunk 4]
```diff
@@ -20,6 +15,15 @@ open http://$(docker-compose port nginx 80)
 
 ## Setup for development
 
+Add core-bundle to the project:
```
Comment (tuj) [inline README-develop.md:18]: Please add comment about not committing composer.json after this change has been made.

**Review by tuj — CHANGES_REQUESTED** [review]

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**